### PR TITLE
Add submission_id to reception event message

### PR DESCRIPTION
**Pull request by harrietc52**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -72,10 +71,9 @@
 
   config.stamp_url = 'http://localhost:7000/api/v1/'
 
-  config.pmb_uri = ENV.fetch('PMB_URI','http://localhost:10000/v1')
-
   config.ehmdmc_url = 'http://localhost:3501/validate'
   config.ehmdmc_url_default_proxy = 'http://localhost:3501'
+  config.pmb_uri = ENV.fetch('PMB_URI', 'http://localhost:3000/v1')
```
Comment (NikolausC) [inline config/environments/development.rb]: Might be worth reverting this as it's on the default rails port

File: spec/lib/event_publisher_spec.rb [hunk 2]
```diff
@@ -29,78 +39,89 @@ def mock_connection(params)
     allow(@channel).to receive(:fanout).and_return(@exchange)
     allow(@exchange).to receive(:name).and_return('exchange name')
 
+    allow(@queue).to receive(:bind)
+    # allow(@queue).to receive(:name).and_return('queue name')
```
Comment (NikolausC) [inline spec/lib/event_publisher_spec.rb]: Could this be removed? Or commented to say what it's waiting on/why it isn't working

File: spec/lib/event_publisher_spec.rb [hunk 3]
```diff
@@ -29,78 +39,89 @@ def mock_connection(params)
...
+    context 'unconfirmed set is not empty' do
+      before(:each) do
+        @unconfirmed_sets = Set.new([1])
+      end
 
-      ep = EventPublisher.new(@params)
-      expect(@exchange).to receive(:publish).with('message', routing_key: @params[:queue_name])
-      expect{ep.publish(@event_message)}.to raise_error(/unconfirmed/)
+      it 'raises exception if unconfirmed set is not empty' do
+        allow(@channel).to receive(:unconfirmed_set).and_return(@unconfirmed_sets)
```
Comment (NikolausC) [inline spec/lib/event_publisher_spec.rb]: Could lines 119-122 be moved into the before(:each)? (Same with 105-108)
Alternatively you could have it run before each of the contexts, then set (let) the unconfirmed_sets within each context. If I remember correctly it'd jump into the current context to get the "let" value, then use it to run the "before" code

File: db/schema.rb [hunk 4]
```diff
@@ -48,7 +48,6 @@
     t.string   "description"
     t.datetime "created_at",                    null: false
     t.datetime "updated_at",                    null: false
```
Comment (NikolausC) [inline db/schema.rb]: The schema you've added here is different as it's missing the boolean "decapper" from "labware_types", but there is no migration in this PR to get rid of it

File: spec/lib/event_publisher_spec.rb (hunk 3 above)
Comment (pjvv) [reply spec/lib/event_publisher_spec.rb]: Next time sunshine.

**Pull request merged**